installer/client/src/pages/ConfirmationScreen.vue

```diff
@@ -0,0 +1,141 @@
+<!--
+  - OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+  - all the essential functionalities required for any enterprise.
+  - Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+  -
+  - OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+  - the GNU General Public License as published by the Free Software Foundation; either
+  - version 2 of the License, or (at your option) any later version.
+  -
+  - OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+  - without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  - See the GNU General Public License for more details.
+  -
+  - You should have received a copy of the GNU General Public License along with this program;
+  - if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+  - Boston, MA  02110-1301, USA
+  -->
+<template>
+  <oxd-form class="orangehrm-installer-page" :loading="isLoading">
+    <oxd-text tag="h5" class="orangehrm-installer-page-title">
+      Confirmation
+    </oxd-text>
+    <br />
+    <oxd-text tag="p" class="orangehrm-installer-page-content">
+      All the information required for OrangeHRM installation collected in the
+      earlier steps are given below. On confirmation the installer will create
+      the database, database users, configuration file, etc
+    </oxd-text>
+    <br />
+    <oxd-text tag="h6" class="confirmation-text-header">Details</oxd-text>
```

Comment:
use `h5` the size will come from it

Reply:
changed. but it seems too big compared to spec